toolforge/deploy_scripts/update_pybot_local.sh

```
@@ -0,0 +1,30 @@
+# Ensure the Python3 binary exists before compiling
+export PYTHON_BIN="$HOME/local/bin/python3"
+export COMPILE_PYTHON_FILES=1
+
+# additional file to copy to TARGET_DIR
+export COPY_TO_TARGET=""
+
+TARGET_DIR="pybot"
+
+# Execute the local deployment script directly without cloning from GitHub
+# We copy the script to ~/deploy_scripts/ first from the github action workflow, then call it
+$HOME/deploy_scripts/deploy_from_local.sh "$REPO_TMP_DIR" "$TARGET_DIR"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)